daskernetes/__init__.py

```diff
@@ -147,17 +137,15 @@ def main():
 
     args = argparser.parse_args()
 
-    scheduler = Scheduler()
-    scheduler.start(('0.0.0.0', '8786'))
     cluster = KubeCluster(
         args.name,
         args.namespace,
         args.worker_image,
         {},
-        scheduler.address
+        n_workers=1,
     )
-    adapative_cluster = Adaptive(scheduler, cluster)
-    tornado.ioloop.IOLoop.current().start()
+    client = Client(cluster)
+    print(client.submit(lambda x: x + 1, 10).result())
```

Comment:
This is just here for testing at the moment.  We should probably set up a CI system with minikube at some point.  I ran into https://blog.travis-ci.com/2017-10-26-running-kubernetes-on-travis-ci-with-minikube .  Other suggestions welcome.